server/src/models/index.js

```diff
@@ -27,7 +27,7 @@ const sub_module_defs = _.compact([
   // pses,
   // sobjs,
   // search,
-  // people,
+  people,
```

Comment:
Right before merging, it'll be worth commenting this back out just so it's clear it's not in use on master